iib/web/app.py

```diff
@@ -38,6 +38,47 @@ def load_config(app):
         app.config.from_pyfile(config_file)
 
 
+def validate_api_config(config):
+    """
+    Determine if the configuration is valid.
+
+    :param dict config: the dict containing the IIB REST API config
+    :raises ConfigError: if the config is invalid
+    """
+    if config['IIB_GREENWAVE_CONFIG']:
+        defined_queue_names = [queue_name for queue_name in config['IIB_USER_TO_QUEUE'].values()]
+        for queue_name in config['IIB_GREENWAVE_CONFIG'].keys():
+            if queue_name not in defined_queue_names:
+                raise ConfigError(
+                    f'Invalid queue name "{queue_name}" defined in "IIB_GREENWAVE_CONFIG"'
```

Comment:
You should also support the default queue by having the key either be `"default"` or `None` in `IIB_GREENWAVE_CONFIG`.

Reply:
I took the way of defining the default queue configuration to be defined by the `None` key. Since that is what the function `_get_user_queue()` returns when the default queue is used.